test/unit/rules/no-yield-block-params-to-else-inverse-test.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE = require('../../../lib/rules/no-yield-block-params-to-else-inverse')
+  .ERROR_MESSAGE;
+
+generateRuleTests({
+  name: 'no-yield-block-params-to-else-inverse',
+
+  config: true,
+
+  good: [
+    '{{yield}}',
+    '{{yield to="whatever"}}',
+    '{{yield to=this.someValue}}',
+    '{{yield to=(get some this.map)}}',
```

Comment:
✂️ these are invalid – your rule should work with them (i.e. you should make sure it does not runtime error when encountering these "unexpected" cases), but they shouldn't be documented as "good" for the purpose of this rule.

Reply:
I see, nvm, ignore me, this is the test, not the documentation.